src/ui/connectiondialog.cc

```diff
@@ -173,7 +160,78 @@ void ConnectionDialog::serverFileSelected(const QString& file_name) {
   ui_->server_executable_line_edit->setText(file_name);
 }
 
+void ConnectionDialog::loadDefaultValues() {
+  ui_->new_server_radio_button->setChecked(
+      util::settings::connection::runServerDefault());
+  serverLocalhost();
+  ui_->port_spin_box->setValue(
+      util::settings::connection::serverPortDefault());
+  ui_->key_line_edit->setText("");
+  clientLocalhost();
+  ui_->client_name_line_edit->setText(
+          util::settings::connection::clientNameDefault());
+  ui_->database_line_edit->setText(
+      util::settings::connection::databaseNameDefault());
+  ui_->server_executable_line_edit->setText("");
```

Comment:
Why for some values we use xxxDefault function from settings, while others have their defaults stored here?